corehq/apps/users/role_utils.py

```diff
@@ -1,4 +1,32 @@
-from corehq.apps.users.models import UserRole, UserRolePresets
+from corehq.apps.users.models import UserRole, Permissions
+
+
+class UserRolePresets:
```

Comment:
This seems like a much nicer/cleaner way to structure this, getting rid of `get_permissions` and `get_preset_map` etc. I'm curious if there's some standard for having it in `utils` vs `models`?

Reply:
Thanks, yeah I was happy to be able to clean it up a bit.

I moved it out of models because `UserRolePresets` is only used in this `role_utils` file and in tests, so it's nice to keep it local.  Also, models files should be focused on db models, which this isn't, and the files (especially `users/models.py`) tend to get really massive and unwieldly, so it's nice to break things out of them when it is clean to do so.